metadata-ingestion/tests/integration/delta_lake/sources/local/table_with_string_and_array.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "delta-lake",
+  "config": {
+    "env": "UAT",
+    "base_path": "tests/integration/delta_lake/test_data/delta_tables_nested_datatype/table_with_string_and_array",
```

Comment:
Instead of 3 separate tests, can we add a single test with `tests/integration/delta_lake/test_data/delta_tables_nested_datatype` as base_path ?

Reply:
will modify accordingly